README.md

```diff
@@ -1,15 +1,18 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Portfolio: setup and mobile version skeleton
+
+> For the first milestone in the process of creating your portfolio website, you will:
+
+    set up a new repository and prepare it for development using best practices (e.g. linters).
+    build the first 2 sections of the mobile website using the template you chose in the previous step.
```

Comment:
- [ ] [OPTIONAL] I think you need to write a description about your project here. This is an instruction from your student dashboard. Please write something about your project like what it is, what it does.